aospy/data_loader.py

```diff
@@ -10,6 +10,26 @@
 from .utils import times, io
 
 
+def preprocess_and_rename_grid_attrs(func):
```

Comment:
Also maybe add 1 sentence to doc explaining why this is necessary, namely to generate single function that gets passed to 'preprocess' within open_mfdataset